R/chk-complex.R

```diff
@@ -0,0 +1,41 @@
+#' Check Complex
+#'
+#' @description
+#' Checks if complex using
+#'
+#' `is.complex(x)`
+#'
+#' @inheritParams params
+#' @inherit params return
+#'
+#' @family data_type_checkers
+#'
+#' @seealso [is.complex()]
+#' @seealso For more details about the use of this function,
+#' please read the article
+#' \href{https://poissonconsulting.github.io/chk/articles/chk-families.html}{chk families}.
+#'
+#' @examples
+#' # chk_complex
+#' chk_complex(as.complex(1))
```

Comment:
I think it would be good to have an additional example here for an actual complex number not just one converted with the `as.complex()` function. As it took me extra time to think wait 1 isn't a complex number shouldn't it fail and then realize it passed because the 1 is wrapped in the `as.complex()` function. 

```suggestion
#' chk_complex(1i)
```